src/syscall/libc_wasip2.go

```diff
@@ -5,16 +5,29 @@
 package syscall
 
 import (
-	"internal/wasm/cm"
 	"strings"
 	"unsafe"
+
+	"github.com/ydnar/wasm-tools-go/cm"
```

Comment:
I'm still getting build errors related to these packages when trying to build my sample program:
```
+ tinygo build -x -target=wasip2 -o main.wasm main.go
/Users/dgryski/Library/Caches/tinygo/goroot-561a20002e9d8c73ac440cca407183cb88c82ed7a444b7ce4e6166f15d2aed51/src/runtime/runtime_tinygowasmp2.go:6:2: no required module provides package github.com/ydnar/wasm-tools-go/cm; to add it:
	go get github.com/ydnar/wasm-tools-go/cm
/Users/dgryski/Library/Caches/tinygo/goroot-561a20002e9d8c73ac440cca407183cb88c82ed7a444b7ce4e6166f15d2aed51/src/runtime/runtime_wasm_wasip2.go:8:2: no required module provides package github.com/ydnar/wasm-tools-go/wasi/cli/environment; to add it:
	go get github.com/ydnar/wasm-tools-go/wasi/cli/environment
...

```

Reply:
I just tried this on a different machine, and it worked. Is it possible your TinyGo src directory != your TINYGOROOT?